src/components/application_manager/test/commands/mobile/set_global_properties_test.cc

```diff
@@ -632,17 +631,10 @@ TEST_F(SetGlobalPropertiesRequestTest, Run_NoVRNoDataNoDefault_Canceled) {
       .WillOnce(Return(mock_app_));
   EXPECT_CALL(mock_message_helper_, VerifyImageVrHelpItems(_, _, _)).Times(0);
   EXPECT_CALL(app_mngr_, RemoveAppFromTTSGlobalPropertiesList(kConnectionKey));
-  SmartObject vr_help_title(smart_objects::SmartType_Null);
-  EXPECT_CALL(*mock_app_, vr_help_title())
-      .WillOnce(Return(&vr_help_title))
-      .WillOnce(Return(&vr_help_title));
-
-  CommandsMap commands_map;
-  DataAccessor<CommandsMap> accessor(commands_map, lock_);
-  EXPECT_CALL(*mock_app_, commands_map()).WillOnce(Return(accessor));
-  const CustomString name("name");
-  EXPECT_CALL(*mock_app_, name()).WillOnce(ReturnRef(name));
-  EXPECT_CALL(*mock_app_, set_vr_help_title(SmartObject(name)));
+  EXPECT_CALL(*mock_app_, vr_help_title()).Times(0);
```

Comment:
@SNiukalov you removed this functionality from implementation so you could remove these expectations from unit test as well